index.html

```diff
@@ -107,11 +112,11 @@ <h3 class="subtitle">Multi-Post Stories Gain+Glory</h3>
         </section>
     </section>
     
-    <section class="tittleContainer aboutMe">
+    <section class="aboutMe tittleContainer" id="About">
```

Comment:
- [ ] Nice work implementing the required change. However, the background image of the mobile version is not there. Please fix this little issue to stick to the template.

Figma template

<img width="249" alt="Screen Shot 2022-11-03 at 10 06 37 PM" src="https://user-images.githubusercontent.com/47673311/199864088-119a2ea1-7a44-4f90-8c59-92072dfb3235.png">

Your design

<img width="249" alt="Screen Shot 2022-11-03 at 10 06 13 PM" src="https://user-images.githubusercontent.com/47673311/199864081-41007bf8-3ca8-4349-a74c-908b54471d8b.png">